### PR TITLE
GlusterFS: Fix registry playbook PV creation

### PR DESCRIPTION
**Pull request by jarrpa**
Resolves: https://bugzilla.redhat.com/show_bug.cgi?id=1625090

Signed-off-by: Jose A. Rivera <jarrpa@redhat.com>

### CODE REVIEW

**Comment by jarrpa** [comment]
@michaelgugino Done

File: roles/openshift_facts/defaults/main.yml [hunk 1]
```diff
@@ -87,7 +87,20 @@ openshift_hosted_registry_storage_glusterfs_ips: []
 openshift_hosted_registry_storage_hostpath_path: /var/lib/openshift_volumes
 # Default to ReadWriteOnce if using hostpath, else default to ReadWriteMany
 openshift_hosted_registry_storage_access_modes:
-  - "{{ (openshift_hosted_registry_storage_kind == 'hostpath') | ternary('ReadWriteOnce', 'ReadWriteMany') }}"
+  - "{{ (openshift_hosted_registry_storage_kind | default(none) == 'hostpath') | ternary('ReadWriteOnce', 'ReadWriteMany') }}"
+
+openshift_hosted_registry_glusterfs_namespace: "{{ openshift_hosted_registry_namespace }}"
+openshift_hosted_registry_glusterfs_storage_kind: 'glusterfs'
```
Comment (michaelgugino) [inline roles/openshift_facts/defaults/main.yml]: Having this default value here will cause it to always be created.

Comment (jarrpa) [reply roles/openshift_facts/defaults/main.yml]: The create_pv and create_pvc variables default to false, which I thought would be sufficient.

**Comment by jarrpa** [comment]
I don't think there's any way to do that. This is to cover the scenario where there is a pre-existing registry NOT using storage_kind=glusterfs and we want to create a new GlusterFS volume to eventually replace the existing volume.

**Comment by michaelgugino** [comment]
@jarrpa is this intending to address some bug or is this just a nice to have?

I'd prefer to put less stuff in that action plugin rather than more.  This is all going away in 4.0, so unless it's a blocker for 3.11 it's not worth doing IMO.

**Comment by jarrpa** [comment]
@michaelgugino This is addressing the bug mentioned in the description. It is not a release blocker, but it must be fixed ASAP in 3.11.z.

**Comment by jarrpa** [comment]
Correct, this should still allow our current support matrix. Installing a first-time registry with storage_kind=glusterfs will work as expected even if only specifying one GlusterFS cluster (either glusterfs or glusterfs_registry) in the inventory file.

**Pull request merged**